occurrences/admin.py

```diff
@@ -123,6 +163,21 @@ class OccurrenceAdmin(admin.ModelAdmin):
     exclude = ("id",)
     list_filter = ["start_time", "end_time", "seat_type", "cancelled"]
     search_fields = ["p_event__linked_event_id"]
+    readonly_fields = ["p_event"]
+    autocomplete_fields = ["contact_persons"]
```

Comment:
**The `p_event` cannot be readonly in `OccurrenceAdmin` or you cannot add a new occurrence!**. Move it to `autocomplete_fields` instead.

```suggestion
    autocomplete_fields = ["p_event", "contact_persons"]
```

Reply:
Fixed as suggested.